<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mystic Square</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="playArea" native="true"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="newGameBtn">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>New Game</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadGameBtn">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Load Game</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveGameBtn">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Save Game</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="moveCountLabel">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewGame"/>
    <addaction name="actionLoadGame"/>
    <addaction name="actionSaveGame"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuBackground">
    <property name="title">
     <string>Background</string>
    </property>
    <addaction name="actionOriginalBackground"/>
    <addaction name="actionForestBackground"/>
    <addaction name="actionTreeBackground"/>
    <addaction name="actionNetworkBackground"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <property name="icon">
     <iconset theme="help-about-symbolic"/>
    </property>
    <addaction name="actionAboutApp"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBackground"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewGame">
   <property name="icon">
    <iconset theme="document-new-symbolic"/>
   </property>
   <property name="text">
    <string>New Game</string>
   </property>
   <property name="toolTip">
    <string>Start a new game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionLoadGame">
   <property name="icon">
    <iconset theme="document-open-symbolic"/>
   </property>
   <property name="text">
    <string>Load Game</string>
   </property>
   <property name="toolTip">
    <string>Load a game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveGame">
   <property name="icon">
    <iconset theme="document-save-symbolic"/>
   </property>
   <property name="text">
    <string>Save Game</string>
   </property>
   <property name="toolTip">
    <string>Save the current game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit-symbolic"/>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOriginalBackground">
   <property name="text">
    <string>Original</string>
   </property>
   <property name="toolTip">
    <string>Load the default background</string>
   </property>
  </action>
  <action name="actionForestBackground">
   <property name="text">
    <string>Forest</string>
   </property>
   <property name="toolTip">
    <string>Load a picture of a forest as the background</string>
   </property>
  </action>
  <action name="actionTreeBackground">
   <property name="text">
    <string>Tree</string>
   </property>
   <property name="toolTip">
    <string>Load a picture of a tree as a background</string>
   </property>
  </action>
  <action name="actionNetworkBackground">
   <property name="text">
    <string>Network</string>
   </property>
   <property name="toolTip">
    <string>Load a picture of a network as a background</string>
   </property>
  </action>
  <action name="actionAboutApp">
   <property name="icon">
    <iconset theme="help-about-symbolic"/>
   </property>
   <property name="text">
    <string>About Mystic Square</string>
   </property>
   <property name="toolTip">
    <string>Learn more about Mystic Square</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset theme="qt"/>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="toolTip">
    <string>Learn more about Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
